src/bin/kickstart.rs

```diff
@@ -2,52 +2,53 @@ use std::env;
 use std::error::Error;
 use std::path::Path;
 
-use clap::{crate_authors, crate_description, crate_version, App, AppSettings, Arg, SubCommand};
+use clap::{arg, command, Command};
 
 use kickstart::generation::Template;
 use kickstart::terminal;
 use kickstart::validate::validate_file;
 
-pub fn build_cli() -> App<'static, 'static> {
-    App::new("kickstart")
-        .version(crate_version!())
-        .author(crate_authors!())
-        .about(crate_description!())
-        .setting(AppSettings::SubcommandsNegateReqs)
+pub fn build_cli() -> Command {
+    command!()
```

Comment:
Can we switch to using the struct way instead of the macro?

Reply:
Sure, there are also a few other issues I am addressing.